logbook-servlet/src/test/java/org/zalando/logbook/servlet/MimeTypesTest.java

```diff
@@ -2,14 +2,14 @@
 
 import org.junit.jupiter.api.Test;
 
-import javax.activation.MimeTypeParseException;
+import java.util.Optional;
 
-import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
 final class MimeTypesTest {
 
     @Test
     void shouldFailToParse() {
-        assertThrows(MimeTypeParseException.class, () -> MimeTypes.parse(""));
+        assertEquals(MimeTypes.parse(""), Optional.empty());
```

Comment:
No happy case test?

Reply:
The happy case is tested within the interceptor test

Reply:
Done